Eask

```diff
@@ -1,3 +1,5 @@
+;; -*- mode: eask; lexical-binding: t -*-
```

Comment:
What does `mode: eask` mean?

Reply:
It will automatically load [eask-mode](https://github.com/emacs-eask/eask-mode) as your major mode when you open this file.

Reply:
Sorry I missed the fact that this modified file is an Eask file :smile: Perfect and thanks!

Reply:
No problem! :)